### ISSUE TITLE
Static link in code?

### ISSUE THREAD

**Issue by Anachron** [hmngwy/jenny#3]
I was considering this static blog generator but it kind of disturbs me that there is this link:
http://manilafunctional.com
all over the code.

Why not use an ENV-variable (like $HOST) for this?

I don't want to clean all of the source for myself before compiling it.

**Comment by hmngwy** [comment]
@Anachron 

Fixed in the above commits. You'll have to add `BLOG_HOST=""` and `BLOG_TITLE=""` in your `.blogrc` file. 

I overlooked those lines, it's a mistake. 

Also, I'd recommend you copy the share folder somewhere, edit the templates as you wish, and point your `.blogrc`'s layout directory var like so `LAYOUT_DIR="../share_copy"`. That way any breaking or dumb commits I make in the template don't get into your site after you upgrade your clone of `jenny`.
